pom.xml

```diff
@@ -36,8 +36,20 @@
       Only update this when releasing a new UI pack.
     -->
     <ui.pack.version>4.3.4_p7</ui.pack.version>
+    <guava.version>27.0.1-jre</guava.version>
```

Comment:
I am not sure if it is right to change guava version like that, 
cdap uses different guava version - https://github.com/cdapio/cdap/blob/develop/pom.xml#L134

@tivv wdyt?
@jockech did some workaround to use suitable guava version required for e2e-tests in this commit - https://github.com/data-integrations/amazon-s3-plugins/commit/d176ef9a8149791f652f4ea2b7a85c61ceeef0e0

Reply:
This workaround was suggested by @tivv as Selenium complains about guava 13.0.1. I'm thinking with the removal of `cdap` as parent in my follow-up PR (will be merged after this one is merged), I can remove guava from this `pom.xml` file altogether.

Reply:
can we add it only in the `e2e-tests` profile?

Reply:
yes, we most probably can. I'll add it to my other PR though as that's where these tests will start running in automation.

Reply:
If we are removing cdap as parent in follow up PR, removing guava altogether from pom sounds good.
Then we don't even need it in `e2e-tests` profile.

Reply:
Actually I take that back, UI uses guava here:
https://github.com/cdapio/cdap-ui/blob/3c7f843ab4bc19585999e0adeef3a2de0058349a/src/main/java/io/cdap/cdap/ui/ConfigurationJsonTool.java#L19

Reply:
Ahh I see, then I would suggest doing something what @jockech did. Adding `v13.0.1` as direct dependency and override it in `e2e-tests` profile with `27.0.1-jre`.

Reply:
Yup, trying that right now

Reply:
@itsankit-google on second thought, why would it hurt to set the guava version to 27? The only change with that happens here and UI uses guava very limitedly
https://github.com/cdapio/cdap-ui/blob/3c7f843ab4bc19585999e0adeef3a2de0058349a/src/main/java/io/cdap/cdap/ui/ConfigurationJsonTool.java#L72

Reply:
> Ahh I see, then I would suggest doing something similar what @jockech did. Adding `v13.0.1` as direct dependency and override it in `e2e-tests` profile with `27.0.1-jre`.

yes. it seems the `profile` can only override a explicitly declared dependency in the POM.

Reply:
You should be able to use dependency management in the profile.

Reply:
jockech@ approach doesn't work in the UI repo, running the `mvn clean verify -P e2e-tests` errors out with
![Screen Shot 2022-07-29 at 8 41 53 PM](https://user-images.githubusercontent.com/94018249/181863610-91aa74ad-70ba-4abc-883f-c9baefe4e180.png)
The only way I've managed to make it work is what I currently have in this PR which Vitalli and I came up with. I'd like to know what's the problem considering there'll be no dependency on cdap eventually which uses a different guava version.